util/memory_access_tracker/test_memory_tracker.c

```diff
@@ -0,0 +1,62 @@
+#include <unistd.h>
```

Comment:
Above this line, please add a copyright statement to formally declare this open source with you as the author. You only nejed this for the .c and .h files. For a model, see:
`util/gdb-dmtcp-utils`